packages/tupaia-web/src/features/Dashboard/Dashboard.tsx

```diff
@@ -25,12 +27,15 @@ const Panel = styled.div<{
 }>`
   position: relative;
   background-color: ${({ theme }) => theme.panel.background};
-  transition: width 0.5s ease, max-width 0.5s ease;
+  transition: width 0.3s ease, max-width 0.3s ease;
   width: 100%;
   overflow: visible;
   min-height: 100%;
   @media screen and (min-width: ${MOBILE_BREAKPOINT}) {
-    width: ${({ $isExpanded }) => ($isExpanded ? 55 : 30)}%;
+    width: ${({ $isExpanded }) =>
+      $isExpanded
+        ? 100
+        : 30}%; // setting this to 100% when expanded takes up approx 50% of the screen, because the map is also set to 100%
```

Comment:
I reckon this is a bit confusing and also a bit limited since the match tupaia the map should be taking up more than half the screen on some widths. I think it would be better to use `flex: 1` on the map instead of `width: 100%` and then this could be put back to using width predictably.